### DISCUSSION TITLE
Issue with drop-migrations-table

### DISCUSSION THREAD

**Discussion by mandyMooreFan** [mikro-orm/mikro-orm#4512] [Q&A]
Currently having an issue with the `schema:drop --drop-migrations-table`. I am running the command and seeing my schema drop, but the migrations table is still populated with old migrations and present. I've also tried `schema:update`.

### Running: 
```
    "@mikro-orm/cli": "5.7.12",
    "@mikro-orm/core": "5.7.12",
    "@mikro-orm/migrations": "5.7.12",
    "@mikro-orm/migrations-mongodb": "5.7.12",
    "@mikro-orm/mongodb": "5.7.12",
    "@mikro-orm/nestjs": "5.2.0",
    "@mikro-orm/seeder": "5.7.12",
```

`mikro-orm schema:drop --drop-migrations-table -r`

### Results:
I see my schema dropped and see the process finish without errors:
`Schema successfully dropped`

But my migrations table is still present with the old migrations that I wanted to drop.

### Issue:
My migrations table is still present with all migrations still in it. In my case the schema:drop runs, but the `--drop-migrations-table` flag isn't working as I expect it to. Not sure what configurations are important to share and will update my question with any needed configs.

**Comment by B4nan** [comment]
Looks like this is not implemented for mongo.

#4513
